src/show/pages/public/Register/index.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+
+import Paths from 'process/routes/paths';
+import { register } from 'process/slices/userSlice';
+import { Button, TextField } from 'show/components';
+
+const CreateAccount = () => {
+  const [user, setUser] = useState({
+    email: '',
+    password: '',
+    first_name: '',
+    last_name: ''
+  });
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const registerUser = e => {
+    e.preventDefault();
+    dispatch(register(user), navigate(Paths.public.LOGIN_PATH));
+  };
+
+  return (
+    <div className='flex w-full h-full items-center bg-primary justify-center'>
+      <div className='flex flex-col w-2/5 h-3/4 bg-white rounded-lg items-center justify-center'>
+        <div className='flex mt-10 mb-20 text-charade text-3xl font-extrabold'>
+          Create Account
+        </div>
+        <div className='flex flex-col w-3/4 h-full justify-center space-y-5'>
+          <div>
+            <TextField
+              label='First Name:'
+              type='text'
+              placeholder='First Name'
+              value={user.first_name}
+              onChange={e => setUser({ ...user, first_name: e.target.value })}
```

Comment:
Instead of this you should be building the user object dynamically by passing the label (e.g. first_name, last_name, email etc.) and the value. 

const onFieldChange = (key, value) => {
... build here
}

so you can use it like this

onChange={e => onFieldChange('first_name', e.target.value)} - for the first_name text field

Reply:
done